### ISSUE TITLE
Broader triggers to cover more use cases

### ISSUE THREAD

**Issue by htang014** [nmevenkamp/windows_wallpaper_scheduler#3]
I think a few additional triggers on scheduled tasks might be useful
* For `WallpaperRefresh_Period_<USER>`:
  * Trigger[ on system time change](https://learn.microsoft.com/en-us/previous-versions/windows/it-pro/windows-10/security/threat-protection/auditing/event-4616) (Security > Microsoft windows security auditing > 4616)
* For `WallpaperRefreshDawnDusk_LOGON_<USER>`:
  * Combine with `WallpaperRefresh_LOGON_<USER>`, i.e. run `refresh_wallpaper.ps1` action sequentially after `refresh_dawn_dusk_times.ps1`
  * Trigger at midnight daily (synchronized across time zones)
  * Trigger on workstation unlock of any user

**Comment by nmevenkamp** [comment]
I agree that the suggested triggers might be useful. Would you be willing to propose according commits?

**Comment by htang014** [comment]
I can in a while.  Trying to see if its possible to use `schtasks /xml` since the inline command doesnt allow multiple triggers but the install script asks for the user password when I do.

**Comment by htang014** [comment]
If I'm looking at this correctly, the `/RU` parameter for schtasks was just passing in the current user and was redundant?  It was causing a password prompt to appear when using `/XML`, so I removed it in the PR
